explorer/src/utils/name-service.tsx

```diff
@@ -81,7 +81,7 @@ export async function performReverseLookup(
       const nameLength = new BN(name!.data.slice(0, 4), "le").toNumber();
       return {
         name: name.data.slice(4, 4 + nameLength).toString() + ".sol",
-        address: name.address,
+        // address: name.address,
```

Comment:
@devstar0718 - is this meant to be commented out?

Reply:
![image](https://user-images.githubusercontent.com/58328476/147495699-5a9a390b-e716-4afd-b4d0-414c5449d6b3.png)
Compile errors with this value because address is not contained by name type.
I am sure that it's never referred by other codes.